src/app/item-popup/DesktopItemActions.tsx

```diff
@@ -120,7 +129,18 @@ export default function DesktopItemActions({ item }: { item: DimItem }) {
 
   return (
     <>
-      <div className={styles.interaction} ref={containerRef}>
+      <div
+        className={clsx(styles.interaction, { [styles.collapsed]: isCollapsed })}
+        ref={containerRef}
+      >
+        <div
+          className={styles.collapseButton}
+          onClick={onToggleSidecar}
+          role="button"
+          tabIndex={-1}
+        >
+          <AppIcon icon={isCollapsed ? faExpandAlt : closeIcon} />
```

Comment:
I had this as a » and « button for expand/contract.

Reply:
That text or a corresponding font-awesome icon? `chevron-double-left` looks to be pro only

Reply:
I think I had something similar from font-awesome but I can't quite remember.

Reply:
a 'tricky' thing about the » « is that depending on what side the sidecar is on, we'd potentially need to flip them?

Reply:
https://fontawesome.com/icons/angle-double-right?style=solid
https://fontawesome.com/icons/angle-double-left?style=solid

Reply:
Yeah you'd definitely need to flip them. `transform: scaleY(-1)`?